.eslintcache

```diff
@@ -0,0 +1 @@
ev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\dashboard\\components\\navbar\\Navbar.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\dashboard\\components\\Dashboard.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\dashboard\\components\\Messages.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\dashboard\\components\\Profile.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\dashboard\\components\\sidebar\\Sidebar.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\ProfileContainer.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\Profile.jsx",["131"],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\dashboard\\components\\sidebar-icons\\DashboardIcon.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\dashboard\\components\\sidebar-icons\\ProfileIcon.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\dashboard\\components\\sidebar-icons\\MessageIcon.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\UserInfo.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\MentorButton.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\UserDescription.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\Rating.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\AboutMe.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\Reviews.jsx",["132","133"],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\Goals.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\RatingComments.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\DropdownButton.jsx",[],"D:\\Web Dev\\Mentorpaired\\frontend_staging\\frontend\\src\\modules\\profile\\components\\Dropdown.jsx",[],{"ruleId":"134","severity":1,"message":"135","line":6,"column":8,"nodeType":"136","messageId":"137","endLine":6,"endColumn":15},{"ruleId":"134","severity":1,"message":"138","line":2,"column":8,"nodeType":"136","messageId":"137","endLine":2,"endColumn":20},{"ruleId":"134","severity":1,"message":"139","line":3,"column":8,"nodeType":"136","messageId":"137","endLine":3,"endColumn":18},{"ruleId":"134","severity":1,"message":"140","line":4,"column":8,"nodeType":"136","messageId":"137","endLine":4,"endColumn":22},"no-unused-vars","'Profile' is defined but never used.","Identifier","unusedVar","'MentorButton' is defined but never used.","'filterIcon' is defined but never used.","'DropdownButton' is defined but never used."]
```

Comment:
I don't seem to understand the purpose of this file 🤔

Reply:
I think it should be added to `.gitignore`.